mava/systems/tf/maddpg/networks.py

```diff
@@ -145,15 +159,22 @@ def make_default_networks(
         policy_network = policy_network_func(policy_network)
 
         # Create the critic network.
-        critic_network = snt.Sequential(
-            [
-                # The multiplexer concatenates the observations/actions.
-                networks.CriticMultiplexer(),
-                networks.LayerNormMLP(
-                    list(critic_networks_layer_sizes[key]) + [1], activate_final=False
-                ),
-            ]
-        )
+        critic_network = [
+            # The multiplexer concatenates the observations/actions.
+            networks.CriticMultiplexer(),
+            networks.LayerNormMLP(
+                list(critic_networks_layer_sizes[key]) + [1],
```

Comment:
Should the + [1] be here in the case of MAD4PG? Probably not right?

Reply:
Combining the networks for MADDPG and MAD4PG looks great thanks @KaleabTessera :raised_hands:

Reply:
Ah yes, that is a bug, thanks @DriesSmit !